servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -502,6 +508,33 @@ public long createFile(TachyonURI path, long blockSizeBytes, boolean recursive,
   }
 
   /**
+   * Reinitializes the blocks of an existing open file.
+   *
+   * @param path the path to the file
+   * @param blockSizeBytes the new block size
+   * @param ttl the ttl
+   * @return the file id
+   * @throws InvalidPathException
+   */
+  public long reinitializeBlock(TachyonURI path, long blockSizeBytes, long ttl)
```

Comment:
Is the name `reinitializeBlock` still descriptive, since you are also resetting ttl in addition to block size?